docs/content/en/docs/tasks/cluster/cluster-terraform.md

```diff
@@ -116,6 +116,101 @@ how to scale your EKS Anywhere worker nodes using the Terraform Kubernetes provi
    kubectl get nodes
    ```
 
+## Manage separate workload clusters using Terraform
+
+Follow these steps if you want to use your initial cluster to create and manage separate workload clusters via Terraform.
+
+### Prerequisites
+- An existing EKS Anywhere cluster imported into Terraform state.
+  If your existing cluster is not yet imported, see this [guide.]({{< relref "#guide" >}}).
+- A cluster configuration file for your new workload cluster.
+  
+### Create cluster using Terraform
+
+1. Create the new cluster configuration Terraform file. 
+   ```bash
+      tfk8s -f new-workload-cluster.yaml -o new-workload-cluster.tf
+   ```
+
+   > **NOTE**: Specify the `namespace` for all EKS Anywhere objects when you are using Terraform to manage your clusters (even for the `default` namespace, use `"namespace" = "default"` on those objects).
+   > 
+   > Ensure workload cluster object names are distinct from management cluster object names. Be sure to set the `managementCluster` field to identify the name of the management cluster.
+
+2. Configure the Terraform cluster resource definition generated in step 1.
+   - Set `metadata.generated` as a [computed field](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest#computed-fields). Add the following to your cluster resource configuration:
+   ```bash
+   computed_fields = ["metadata.generated"]
+   ```
+   - Configure the field manager to [force reconcile managed resources](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest#field_manager). Add the following configuration block to your cluster resource:
```

Comment:
True.  This was added for cases where the user used something like `kubectl edit` to make changes to the cluster. In those cases, trying to use Terraform to manage fields previously altered by something like `kubectl-edit` would result in field manager conflicts. Example error after terraform apply: 

```
│ Error: There was a field manager conflict when trying to apply the manifest for "default/tneylas"
│ 
│   with kubernetes_manifest.cluster_tneylas,
│   on wl.tf line 1, in resource "kubernetes_manifest" "cluster_tneylas":
│    1: resource "kubernetes_manifest" "cluster_tneylas" {
│ 
│ The API returned the following conflict: "Apply failed with 1 conflict: conflict with \"kubectl-edit\" using anywhere.eks.amazonaws.com/v1alpha1: .spec.workerNodeGroupConfigurations"
│ 
│ You can override this conflict by setting "force_conflicts" to true in the "field_manager" block.
╵
```

I removed it for now since I didn't see any difference in behavior with or without setting this when using only Terraform to modify the cluster. If there are ever any cases where the user actually wants to force conflicts after not being able to apply, it will explicitly tell them to add this same block of code to force apply those changes (as seen in the error message). From there, the user is able to make the decision on whether or not to add this as they manage clusters

Reply:
Cool
I would even add a disclaimer in the docs saying that if you are using terraform, dont use kubectl

Reply:
Sure! I'll add that :+1: